aws-bundle/NOTICE

```diff
@@ -5,92 +5,22 @@ Copyright 2017-2024 The Apache Software Foundation
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
 
---------------------------------------------------------------------------------
-
-NOTICE for Group: commons-codec  Name: commons-codec  Version: 1.17.1
```

Comment:
It is not necessary to duplicate the line "This product includes software developed at the Apache Software Foundation...", though the ASF copyright line and any other portions of NOTICE must be considered for propagation. I saw it's still included, which is fine, just wanted to provide some "background".